.github/workflows/test_on_push.yaml

```diff
@@ -13,13 +13,24 @@ jobs:
       matrix:
         # We need 1.10.6 here to check that module works with
         # old Tarantool versions that don't have "tuple-keydef"/"tuple-merger" support.
-        tarantool-version: ["1.10.6", "1.10", "2.2", "2.3", "2.4", "2.5", "2.6", "2.7"]
+        tarantool-version: ["1.10.6", "1.10", "2.2", "2.3", "2.4", "2.5", "2.6", "2.7", "2.8"]
+        metrics-version: [""]
         remove-merger: [false]
+        perf-test: [false]
```

Comment:
Raw idea: it would look better in the jobs list if we'll move it to its own job. It seems, it will require some code duplication, but not much (and it can be eliminated with [reusable workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows)).

Maybe the same worth to apply to coverage collecting as well.

Reply:
Idea implemented